package.json

```diff
@@ -14,40 +14,41 @@
     "copy:test-assets": "cpx \"./src/tests/assets/**/{*,.*}\" ./lib/tests/assets"
   },
   "dependencies": {
-    "@bentley/backend-itwin-client": "^2.17.2",
-    "@bentley/bentleyjs-core": "^2.17.2",
-    "@bentley/ecschema-metadata": "^2.17.2",
-    "@bentley/electron-manager": "^2.17.2",
-    "@bentley/frontend-authorization-client": "^2.17.2",
-    "@bentley/geometry-core": "^2.17.2",
-    "@bentley/imodelhub-client": "^2.17.2",
-    "@bentley/imodeljs-backend": "^2.17.2",
-    "@bentley/imodeljs-common": "^2.17.2",
-    "@bentley/imodeljs-frontend": "^2.17.2",
-    "@bentley/imodeljs-i18n": "^2.17.2",
-    "@bentley/imodeljs-quantity": "^2.17.2",
-    "@bentley/itwin-client": "^2.17.2",
-    "@bentley/orbitgt-core": "^2.17.2",
-    "@bentley/presentation-common": "^2.17.2",
-    "@bentley/product-settings-client": "^2.17.2",
-    "@bentley/rbac-client": "^2.17.2",
-    "@bentley/telemetry-client": "^2.17.2",
-    "@bentley/ui-abstract": "^2.17.2",
-    "@bentley/webgl-compatibility": "^2.17.2",
+    "@itwin/insights-client": "0.4.0",
+    "@bentley/backend-itwin-client": "^2.19.51",
+    "@bentley/bentleyjs-core": "^2.19.51",
+    "@bentley/ecschema-metadata": "^2.19.51",
+    "@bentley/electron-manager": "^2.19.51",
+    "@bentley/frontend-authorization-client": "^2.19.51",
+    "@bentley/geometry-core": "^2.19.51",
+    "@bentley/imodelhub-client": "^2.19.51",
+    "@bentley/imodeljs-backend": "^2.19.51",
+    "@bentley/imodeljs-common": "^2.19.51",
+    "@bentley/imodeljs-frontend": "^2.19.51",
+    "@bentley/imodeljs-i18n": "^2.19.51",
+    "@bentley/imodeljs-quantity": "^2.19.51",
+    "@bentley/itwin-client": "^2.19.51",
+    "@bentley/orbitgt-core": "^2.19.51",
+    "@bentley/presentation-common": "^2.19.51",
+    "@bentley/product-settings-client": "^2.19.51",
+    "@bentley/rbac-client": "^2.19.51",
+    "@bentley/telemetry-client": "^2.19.51",
+    "@bentley/ui-abstract": "^2.19.51",
+    "@bentley/webgl-compatibility": "^2.19.51",
     "electron": "^11.1.0"
   },
   "devDependencies": {
-    "@bentley/build-tools": "^2.17.2",
-    "@bentley/eslint-plugin": "^2.17.2",
-    "@types/chai": "^4.2.21",
-    "@types/mocha": "^8.2.3",
+    "@bentley/build-tools": "2.19.51",
```

Comment:
Upgrade to latest iTwinJs version (3.6.1)

Reply:
This is done, but I do not have a "resolve" button for any of the comments I added :/

Reply:
I'll do it 🚀